examples/README.md

```diff
@@ -1,6 +1,6 @@
 # Commerce SDK Examples
 
-This directory contains example usage of the Commerce SDK. Each example contains valid credentials for a public demo instance. The examples can be executed successfully as-is, or you can edit them to provide your own credentials.
+This directory contains example usage of the Commerce SDK. Each example contains valid credentials for a public demo instance. Examples 01-03 can be executed successfully as-is (or you can edit them to provide your own credentials). Examples 04-06 require additional editing (marked by TODO).
```

Comment:
Maybe mark as `EDIT HERE` or something that's more clear it's for the user to do, not an incomplete example?